components/icon/icons.css

```diff
@@ -28,3 +28,21 @@ governing permissions and limitations under the License.
   /* Don't catch clicks or hover, otherwise they may not escape the SVG */
   pointer-events: none;
 }
+
+.spectrum-Icon {
+  &,
+  img,
```

Comment:
FYI: Nesting the size of `img` and `svg` in the same rules as `.spectrum-Icon` will likely cause issue in code like: https://github.com/adobe/spectrum-css/blob/731aeab0a10355dc274efd19d64a58fe71b2ef87/components/tooltip/index.css#L137-L149

Maybe the goal is to ALWAYS use CSS Custom Properties in this sort of context? In that case we'd need to have any such instance updated as part of this PR or prior to an actual release of these changes.

An alternative would be to not directly size the `img` and `svg` content and set them as:
```
    block-size: 100%;
    inline-size: 100%;
```

Thoughts?